coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -323,7 +324,7 @@ public Response startLRA(
             @QueryParam(PARENT_LRA_PARAM_NAME) @DefaultValue("") String parentLRA,
             @HeaderParam(HttpHeaders.ACCEPT) @DefaultValue(MediaType.TEXT_PLAIN) String mediaType,
             @Parameter(ref = LRAConstants.NARAYANA_LRA_API_VERSION_HEADER_NAME)
-            @HeaderParam(LRAConstants.NARAYANA_LRA_API_VERSION_HEADER_NAME) @DefaultValue(CURRENT_API_VERSION_STRING) String version) throws WebApplicationException {
+            @HeaderParam(LRAConstants.NARAYANA_LRA_API_VERSION_HEADER_NAME) @DefaultValue(CURRENT_API_VERSION_STRING) String version) {
```

Comment:
Removed throwing of WebApplicationException but hopefully it would not cause compilation error as I think WebApplicationException should be a RuntimeException - do you agree?

Reply:
Yes it is a RuntimeException and any JAX RS method can [throw it](https://javadoc.io/doc/javax.ws.rs/javax.ws.rs-api/2.0-rc3/javax/ws/rs/WebApplicationException.html). I should have removed it from all of the coordinator methods. Any actual JAX RS status code wrapped by the WebApplicationException is documented via the [MicroProfile OpenAPI](https://download.eclipse.org/microprofile/microprofile-open-api-1.1.2/microprofile-openapi-spec.html) annotations at the start of each coordinator method.

If you don't mind I will remove them all from the Coordinator.java resource.

Reply:
Actually it was the only place where a signature throws WebApplicationException, lol

Reply:
:D